From dc2a33a0e3eaea4c8b5baca80ab5bcddadf19636 Mon Sep 17 00:00:00 2001
From: Sjoerd Simons <sjoerd.simons@collabora.co.uk>
Date: Wed, 6 May 2015 12:39:14 +0200
Subject: [PATCH] notify test: Don't test object relying on dbus connections

Don't test types that rely on a pre-existing dbus session bus.
---
 testsuite/gtk/notify.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/testsuite/gtk/notify.c b/testsuite/gtk/notify.c
index 002c207..1ae3ab6 100644
--- a/testsuite/gtk/notify.c
+++ b/testsuite/gtk/notify.c
@@ -403,6 +403,10 @@ test_type (gconstpointer data)
       g_type_is_a (type, GTK_TYPE_FILE_CHOOSER_WIDGET) ||
       g_type_is_a (type, GTK_TYPE_PLACES_SIDEBAR))
     return;
+
+  /* These rely on a d-bus session bus */
+  if (g_type_is_a (type, GTK_TYPE_MOUNT_OPERATION))
+    return;
  
   klass = g_type_class_ref (type);
 
-- 
2.1.4

